v2/gcs-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/processing/handler/GCSToSourceStreamingHandler.java

```diff
@@ -89,6 +112,47 @@ public static String process(ProcessingContext taskContext, SpannerDao spannerDa
     return fileProcessedStartInterval;
   }
 
+  private static void writeFilteredEventsToGcs(
```

Comment:
please add tests for this method.

Reply:
Do these methods need to be static ?

Reply:
Added tests. The method is being referenced from a static method, therefore this needs to be static.